libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/MapboxOnboardRouter.kt

```diff
@@ -34,7 +37,8 @@ import kotlinx.coroutines.withContext
 @MapboxModule(MapboxModuleType.NavigationOnboardRouter)
 class MapboxOnboardRouter(
     private val navigatorNative: MapboxNativeNavigator,
-    config: MapboxOnboardRouterConfig
+    config: MapboxOnboardRouterConfig,
+    private val logger: Logger = MapboxLogger
```

Comment:
This shouldn't have the default value, it always has to be passed as an argument.